controllers/resource_helper.go

```diff
@@ -109,6 +110,34 @@ func configurePodStorage(instance *llamav1alpha1.LlamaStackDistribution, contain
 				},
 			},
 		})
+
+		// Add init container to fix permissions on the PVC mount
+		mountPath := llamav1alpha1.DefaultMountPath
+		if instance.Spec.Server.Storage.MountPath != "" {
+			mountPath = instance.Spec.Server.Storage.MountPath
+		}
+
+		initContainer := corev1.Container{
+			Name:  "update-pvc-permissions",
+			Image: "registry.access.redhat.com/ubi9/ubi-minimal:latest",
+			Command: []string{
+				"/bin/sh",
+				"-c",
+				fmt.Sprintf("chown --verbose --recursive 1001:0 %s", mountPath),
```

Comment:
that assumes user 1001 exists in the distribution image :)

Reply:
Now I'm wondering if `chmod` would have been easier, so we don't have to guess the user ID...